src/main/scala/ai/eto/rikai/sql/model/Model.scala

```diff
@@ -16,6 +16,10 @@
 
 package ai.eto.rikai.sql.model
 
+import org.json4s._
+import org.json4s.jackson.Serialization
+import org.json4s.jackson.Serialization.write
```

Comment:
Lets use `circe-json` and add dependencies to `build.sbt` like this:

https://github.com/eto-ai/rikai/pull/88/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R29